platform-includes/migration/javascript-v8/important-changes/javascript.react.mdx

```diff
@@ -5,3 +5,21 @@ Sentry React SDK `8.x` supports React version `16.0.0` or higher.
 If you need to support older versions of React, please use Sentry Next.js SDK `7.x`.
 
 <Include name="migration/javascript-v8/compatible-browsers" />
+
+### Updated error types to be `unknown` instead of `Error`.
+
+In v8, we are changing the `ErrorBoundary` error types returned from `onError`, `onReset`, `onUnmount`, and
+`beforeCapture` to be `unknown` instead of `Error`. This more accurately matches behaviour of `componentDidCatch`, the
+lifecycle method the Sentry `ErrorBoundary` component uses.
```

Comment:
```suggestion
In v8, we are changing the `ErrorBoundary` error types returned from `onError`, `onReset`, `onUnmount`, and
`beforeCapture` to be `unknown` instead of `Error`. This more accurately matches the behavior of `componentDidCatch`, which is the
lifecycle method Sentry's `ErrorBoundary` component uses.
```